recipes/libepoxy/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conans import ConanFile, Meson, tools
+import os
+import shutil
+import glob
+
+
+class EpoxyConan(ConanFile):
+    name = "libepoxy"
+    description = "libepoxy is a library for handling OpenGL function pointer management"
+    topics = ("conan", "libepoxy", "opengl")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/anholt/libepoxy"
+    license = "MIT"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "glx": [True, False],
+        "egl": [True, False],
+        "x11": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "glx": True,
+        "egl": False,
+        "x11": True
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            self.options.shared = True
```

Comment:
```suggestion
        if self.settings.os == "Windows":
            if not self.options.shared:
                raise ConanInvalidConfiguration("Static builds on Windows are not supported")
```

Reply:
combining this with shared=False by default means CCI won't provide any windows binaries for package (gtk), and it's up to every user to set the option eoxy:shared=True.

Reply:
Good defaults are important. So please document them.